### PR TITLE
Skip to content

### PR DESCRIPTION
**Pull request by robertvrabel**
Change the skip to content link to goto the content area instead of the whole panel area. This way when you tab, it'll tab into the actual content area.

### CODE REVIEW

File: resources/views/components/content-area.blade.php [hunk 1]
```diff
@@ -43,7 +43,7 @@
             </nav>
         </div>
 
-    <div class="w-full{{ !in_array($page['controller'], config('base.full_width_controllers')) ? ' px-4' : '' }} {{$show_site_menu === true ? 'mt:w-3/4' : '' }} content-area mb-8">
+    <div class="w-full{{ !in_array($page['controller'], config('base.full_width_controllers')) ? ' px-4' : '' }} {{$show_site_menu === true ? 'mt:w-3/4' : '' }} content-area mb-8" id="content">
```
[the remark this answers is not in the document]
Comment (robertvrabel) [reply resources/views/components/content-area.blade.php]: 👍

[the remark this answers is not in the document]
Comment (robertvrabel) [reply resources/views/components/content-area.blade.php]: Hmm, do you think the tabindex should be removed from the main element now that we aren't skipping to it? `<main id="panel" tabindex="-1">`

[the remark this answers is not in the document]
Comment (nickdenardis) [reply resources/views/components/content-area.blade.php]: Yes if it should not longer be focusable when tabbing through the site.

[the remark this answers is not in the document]
Comment (robertvrabel) [reply resources/views/components/content-area.blade.php]: New PR https://github.com/waynestate/base-site/pull/223

**Pull request closed without merging**